README.md

```diff
@@ -2,15 +2,15 @@
 
 This action allows caching dependencies and build outputs to improve workflow execution time.
 
-<a href="https://github.com/actions/cache/actions?query=workflow%3ATests"><img alt="GitHub Actions status" src="https://github.com/actions/cache/workflows/Tests/badge.svg?branch=main&event=push"></a>
+[![Tests](https://github.com/actions/cache/actions/workflows/workflow.yml/badge.svg)](https://github.com/actions/cache/actions/workflows/workflow.yml)
```

Comment:
Why does this require change?

Reply:
Thanks for the review @vsvipul. Since I was in the area of `README.md` - noted this workflow badge was in the "old format" so decided to bolt on this change. 

It's the Markdown badge format generated by the GitHub badge builder:

![Screen Shot 2022-02-28 at 4 02 27 pm](https://user-images.githubusercontent.com/1818757/155926905-843d164b-cd85-4973-814e-9a5b8308bdac.png)

![Screen Shot 2022-02-28 at 4 02 32 pm](https://user-images.githubusercontent.com/1818757/155926900-85d962ad-a05c-4124-a0a3-98141d22ef37.png)

Also of note - the URL format changed when using the build button Markdown UI - it's now tied to the workflow filename (e.g. `workflow.yml` in this instance), rather than `?query=workflow%3ATests`. You'd have to question that with your GitHub Actions team internally, but I guess that feels like a better hook and I tend to agree.

Reply:
![Screen Shot 2022-03-02 at 11 58 11 am](https://user-images.githubusercontent.com/1818757/156274359-b3c81192-98b6-4147-9783-69700dcda0bb.png)

Ah yes, @vsvipul - the other advantage to the updated build badge URL - it now correctly highlights the correct workflow item/name in the sidebar. The old URL `?query=` isn't able to do this.

Reply:
Got it. Thanks for explaining.

Reply:
No dramas @vsvipul 👍 

Hopefully we can get the rest of the PR over the line. 👍